src/test/java/com/google/plantasy/GameTest.java

```diff
@@ -69,12 +68,16 @@ public void SetsGameEntityCorrectly() {
     //Tests if the entity created by the game class gets put into datastore
     public void putInDatastore() {
         Game game = new Game(new Entity("Game"));
-        game.setGameId("1234");
-
+ 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(game.getGameEntity());
 
-        Entity datastore_game_entity = UserUtils.getEntityFromDatastore("Game", "gameId", game.getGameId(), datastore);
-        Assert.assertEquals(game.getGameId(), datastore_game_entity.getProperty("gameId"));
+        Entity datastore_game_entity = null;
+        try{
+            datastore_game_entity = datastore.get(KeyFactory.stringToKey(game.getGameId()));
+        }catch(Exception e){
+            Assert.assertEquals(game.getGameId(), KeyFactory.keyToString(datastore_game_entity.getKey()));
+        }
+        Assert.assertEquals(game.getGameId(), KeyFactory.keyToString(datastore_game_entity.getKey()));
```

Comment:
I'm a bit confused by these two asserts. It looks like they do the same thing. Do we need both of them? Is one more important than the other? Do we expect an exception to be thrown? If we expect an exception, we should test for that explicitly.

Reply:
Switched the first Assert to fail